install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -0,0 +1,222 @@
+----
+====
++
+Choose (1) and follow the on-screen instructions to complete your desired task.
+
+[[adding-nodes-advanced]]
+== Adding Hosts Using the Advanced Install
+
+If you installed using the advanced install, you can add new hosts to your
+cluster by running the *_scaleup.yml_* playbook. This playbook queries the
+master, generates and distributes new certificates for the new hosts, then runs
+the configuration playbooks on the new hosts only. Before running the
+*_scaleup.yml_* playbook, complete all prerequisite
+xref:../install_config/install/prerequisites.html#host-preparation[host
+preparation] steps.
+
+
+ifdef::openshift-enterprise[]
+This process is similar to re-running the installer in the
+xref:adding-nodes-or-reinstalling-quick[quick installation method to add nodes],
+however you have more configuration options available when using the advanced
+method and when running the playbooks directly.
+endif::[]
+
+You must have an existing inventory file (for example, *_/etc/ansible/hosts_*)
+that is representative of your current cluster configuration in order to run the
+*_scaleup.yml_* playbook.
+ifdef::openshift-enterprise[]
+If you previously used the `atomic-openshift-installer` command to run your
+installation, you can check *_~/.config/openshift/.ansible/hosts_* for the last
+inventory file that the installer generated, and use or modify that as needed as
+your inventory file. You must then specify the file location with `-i` when
+calling `ansible-playbook` later.
+endif::[]
+
+[IMPORTANT]
+====
+The recommended maximum number of nodes is 300.
+====
+
+To add a host to an existing cluster:
+
+. Ensure you have the latest playbooks by updating the *atomic-openshift-utils*
+package:
++
+----
+# yum update atomic-openshift-utils
+----
+
+. Edit your *_/etc/ansible/hosts_* file and add *new_<host_type>* to the
+*[OSEv3:children]* section:
++
+For example, to add a new node host, add `new_nodes`:
++
+====
+----
+[OSEv3:children]
+masters
+nodes
+new_nodes
+----
+====
++
+To add new master hosts, add *new_masters*.
+
+. Create a *[new_<host_type>]* section much like an existing section,
+specifying host information for any new hosts you want to add. For example,
+when adding a new node:
++
+====
+----
+[nodes]
+master[1:3].example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
+node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
+
+[new_nodes]
+node3.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
+----
+====
++
+See
+xref:../install_config/install/advanced_install.adoc#advanced-host-variables[Configuring
+Host Variables] for more options.
+
+. Hosts added to the *[new_masters]* section should also be added to the
+ *[new_nodes]* section when running the master *_scaleup.yml_* playbook. For
+ example:
```

Comment:
If this step gets combined w/ the previous step, suggest wording like the following (to match the above wording "when adding a new node"):

"When adding new masters, hosts added to the [new_masters] section must also be added to the [new_nodes] section with the openshift_schedulable=false variable. This ensures the new master host is part of the OpenShift SDN and that pods are not scheduled for placement on them. For example:"